source/object-keys.ts

```diff
@@ -0,0 +1,20 @@
+/**
+A typed implementation of `Object.keys()`.
+
+This is useful since `Object.keys()` returns an array of strings. This function returns an array of the type keys of the given object.
+
+@example
+```ts
+import {objectKeys} from 'ts-extras';
+
+type Item = ['a', 'b', 'c'];
+declare let: items: Item[];
+
+items = objectKeys({a: 1, b: 2, c: 3}); // This is valid.
```

Comment:
I think a better example would just be to do first do `Object.keys` and then `objectKeys` and show the difference in result values.